mindee/cli.py

```diff
@@ -131,6 +150,58 @@ def call_endpoint(args: Namespace):
         print(parsed_data.document)
 
 
+def process_parse_queued(args: Namespace, client: Client, doc_class) -> None:
+    """Processes the results of a queued parsing request."""
+    input_doc = client.doc_for_async()
+    if args.product_name == "custom":
+        parsed_data = input_doc.parse_queued(
+            document_class=doc_class,
+            queue_id=args.queue_id,
+            endpoint_name=args.api_name,
+            account_name=args.username,
+        )
+    else:
+        parsed_data = input_doc.parse_queued(
+            document_class=doc_class, queue_id=args.queue_id
+        )
+    if parsed_data.job.status == "completed" and parsed_data.document is not None:
+        if args.output_type == "raw":
+            print(json.dumps(parsed_data.document.http_response, indent=2))
+        elif args.output_type == "parsed":
+            doc = parsed_data.document.document
+            print(json.dumps(doc, indent=2, default=serialize_for_json))
+        else:
+            print(parsed_data.document.document)
```

Comment:
Duplicate code, compare with output handling in `process_parse`.

Better to create a function to handle this, this way we can more easily add any output methods.